### PR TITLE
Quick CSS tweaks

### PR DESCRIPTION
**Pull request by asportnoy**
- Catch error if file doesn't exist
- Add button to open folder
- Create file on boot if it doesn't exist already

### CODE REVIEW

File: src/util.ts [hunk 1]
```diff
@@ -32,3 +31,8 @@ export const CONFIG_PATHS = Object.fromEntries(
     return [name, path];
   }),
 ) as Record<typeof CONFIG_FOLDER_NAMES[number], string>;
+
+const QUICK_CSS_FILE = join(CONFIG_PATHS.quickcss, "main.css");
+if (!existsSync(QUICK_CSS_FILE)) {
+  writeFileSync(QUICK_CSS_FILE, "");
+}
```
Comment (12944qwerty) [inline src/util.ts:38]: Shouldn't we create the files for all? I know this pr is for quick css but still :shrug:

File: src/main/ipc/quick-css.ts [hunk 2]
```diff
@@ -6,7 +6,9 @@ import { CONFIG_PATHS } from "src/util";
 
 const CSS_PATH = join(CONFIG_PATHS.quickcss, "main.css");
 
-ipcMain.handle(RepluggedIpcChannels.GET_QUICK_CSS, () => readFile(CSS_PATH, { encoding: "utf-8" }));
+ipcMain.handle(RepluggedIpcChannels.GET_QUICK_CSS, () =>
+  readFile(CSS_PATH, { encoding: "utf-8" }).catch(() => ""),
```
Comment (12944qwerty) [inline src/main/ipc/quick-css.ts:10]: Shouldn't there be better errorhandling? Maybe make the file if it's not found?

File: src/util.ts (hunk 1 above)
Comment (asportnoy) [reply src/util.ts:38]: This is the only file we needed to make. No files to create for plugins and themes and we don't know the file names for settings (plus they'll be created when they're needed).

File: src/main/ipc/quick-css.ts (hunk 2 above)
Comment (asportnoy) [reply src/main/ipc/quick-css.ts:10]: Already added that.

Comment (12944qwerty) [reply src/main/ipc/quick-css.ts:10]: That's on discord load... not here?

Comment (asportnoy) [reply src/main/ipc/quick-css.ts:10]: Discord load happens before this. Adding it here will make no difference.

**Review by 12944qwerty — APPROVED** [review]

**Pull request merged**